packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/AirdropReminder/index.js

```diff
@@ -105,7 +105,13 @@ class AirdropReminder extends React.PureComponent {
               id='modals.airdropreminder.getstarted'
             />
           </FooterButton>
-          <LaterButton nature='primary' size='18px' fullwidth onClick={close}>
+          <LaterButton
+            nature='primary'
+            size='18px'
+            fullwidth
+            onClick={close}
+            data-e2e='remindMeLaterButton'
```

Comment:
You might want to name this something more specific, there are a few of these modals

Reply:
They were named the same in order to close any "get started" modals before executing tests to avoid overlapping with other elements